tools/e2e/runIndividualE2EFiles.ts

```diff
@@ -28,5 +28,6 @@ for (const file of e2eTests) {
 for (const task of tasks.values()) {
 	execSync(task, {
 		stdio: "inherit",
+		env: process.env,
```

Comment:
This is needed so that you can run `pnpm run test:e2e:wrangler` locally while passing environment variables in at the command line. E.g.

```bash
CLOUDFLARE_ACCOUNT_ID=xxxx CLOUDFLARE_API_TOKEN=yyyy  WRANGLER="node $PWD/packages/wrangler/bin/wrangler.js" WRANGLER_IMPORT=$PWD/packages/wrangler/wrangler-dist/cli.js pnpm run test:e2e:wrangler
```

Reply:
Actually this doesn't seem to help 😢 
I'll revert this change.